### DISCUSSION TITLE
RunPythonAsync

### DISCUSSION THREAD

**Discussion by gavinowens** [pyodide/pyodide#1682] [General]
When using RunPythonAsync, it loads the libraries correctly, but the code won't run then. May someone help?

**Comment by hoodmane** [comment]
Can you give an example?

**Reply by gavinowens** [reply]
languageLoaderPlugin.then(async function () {
await runPythonAsync(`
Code
`)
});

I later display the output and it displays an object promise or after all the content.

**Reply by xhluca** [reply]
@Goman60 can you try to run a `print(obj)` inside your python code to see if the code is being run? getting a promise is normal since you are running it asynchronously. Usually when i get a promise in the console i can use `await` to resolve it.
